python/Ganga/Core/GangaRepository/GangaRepositoryXML.py

```diff
@@ -129,9 +129,11 @@ def rmrf(name, count=0):
     if os.path.isdir(name):
 
         try:
-            remove_name = name + "_" + str(time.time()) + '__to_be_deleted_'
-            os.rename(name, remove_name)
-            #logger.debug("Move completed")
+            remove_name = name
+            if not remove_name.endswith('__to_be_deleted'):
+                remove_name = name + "_" + str(time.time()) + '__to_be_deleted_'
```

Comment:
I was only referring to that last line, not the whole block:
```python
remove_name = name
if not remove_name.endswith('__to_be_deleted'):
    remove_name += "_" + str(time.time()) + '__to_be_deleted_'
```
or, even better,
```python
remove_name = name
if not remove_name.endswith('__to_be_deleted'):
    remove_name += "_%s__to_be_deleted_" % time.time()
```